### DISCUSSION TITLE
Ability to change /storage path

### DISCUSSION THREAD

**Discussion by gurgles-1** [RandomNinjaAtk/docker-radarr-extended#31] [General]
I would like to be able to change the /storage in unraid. It appears hardcoded now. I would like to do this because I follow trash guides, and using /storage is confusing to me when I always use /data for my shares per Trash.

**Comment by RandomNinjaAtk** [comment]
You can change it to whatever you want technically, only /config volume is required. Your free to modify the provided template for unraid....

**Comment by gurgles-1** [comment]
The template does not allow editing on the unraid ca apps template, is this a bug? You can't change the container path was my point. Thanks for looking into it, and thanks for creating this container.

**Reply by RandomNinjaAtk** [reply]
https://github.com/RandomNinjaAtk/unraid-templates/commit/21a293bb25e0919be331fed06f4fa6f618e2af3a

**Comment by gurgles-1** [comment]
Thanks for the updates - can you add the data path back into the template but make it editable? Thanks! I can personally just add it myself, but for newer users it would be helpful.
